challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map {|s| s.capitalize}
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select{|s| s == "dog"}
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.reject{|s| s == "dog"}
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.count
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input.take(2)
 end
 
 def fetch_CD_animals(input)
-  #implement your solution here
+  input.select{ |s| s.downcase[0] == "c" || s.downcase[0] == "d" }
```

Comment:
A regex here would simplify things.  Also, don't the rules say you can't repeat a method? For shame!